### PR TITLE
Add external provisioners to PV doc

### PR DESCRIPTION
**Pull request by wongma7**
1.6 docs for https://github.com/kubernetes/features/issues/183
@devin-donnelly 
@jsafrane for tech review if needed

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2843)

### CODE REVIEW

File: docs/user-guide/persistent-volumes/index.md [hunk 1]
```diff
@@ -356,6 +356,16 @@ parameters:
 Storage classes have a provisioner that determines what volume plugin is used
 for provisioning PVs. This field must be specified.
 
+Administrators are not restricted to specifying the "internal" provisioners
+listed here (whose names are prefixed with "kubernetes.io" and shipped
+alongside Kubernetes), they can also run and specify external provisioners
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: "Administrators" are the readers in this case. Replace with "you." As in "You are not restricted to specifying..."

Replace comma with period, start a new sentence. "You can also run and specify external provisioners, which are independent programs..."

File: docs/user-guide/persistent-volumes/index.md [hunk 2]
```diff
@@ -356,6 +356,16 @@ parameters:
...
+which are independent programs that follow a [specification](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/volume-provisioning.md)
+defined by Kubernetes. Authors of external provisioners have full discretion
+over where their code lives, how they're shipped, how they need to be run, what
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: "how they're shipped" -> "how the provisioner is shipped, how it needs to be run, what volume plugin it uses, etc."

**Comment by jsafrane** [comment]
lgtm from tech side.

**Comment by wongma7** [comment]
Changes made, thanks

**Pull request merged**